src/Api/PubnubApi/PNConfiguration.cs

```diff
@@ -9,8 +9,14 @@ public class PNConfiguration
     {
         private int presenceHeartbeatTimeout;
         private int presenceHeartbeatInterval;
-        private string uuid = "";
+        private UserId _userId;
+        private bool uuidSetFromConstructor = false;
 
+        internal bool Validate()
+        {
+            uuidSetFromConstructor = false;
+            return true;
+        }
```

Comment:
Is this function necessary? It always returns true.

Reply:
Refactored the code. Now it is void method  with required LOC.